domain/schema/controller.go

```diff
@@ -272,3 +273,16 @@ BEGIN
 END;
 `[1:]
 }
+
+func modelListSchema() string {
+	return `
+CREATE TABLE model_list (
+    uuid        TEXT PRIMARY KEY,
```

Comment:
I don't think we need the synthetic UUID, when we already have an entity identified by one.
Just have `uuid`, implying the model UUID that we use today.
I also wonder if we might need the model name here in the controller DB.

Reply:
Then we have two places to update for model name... What happens if they go out of sync?

  - models
  - model table